reciprocalspaceship/utils/rfree.py

```diff
@@ -46,7 +52,7 @@ def add_rfree(dataset, fraction=0.05, ccp4_convention=False, inplace=False):
     return dataset
 
 
-def copy_rfree(dataset, dataset_with_rfree, inplace=False):
+def copy_rfree(dataset, dataset_with_rfree, inplace=False, custom_rfree_key=None):
```

Comment:
I think this the parameter should be renamed to `rfree_key` -- I think the "custom" is unnecessary here.